### DISCUSSION TITLE
Filter/Sort Products by Price

### DISCUSSION THREAD

**Discussion by dwene** [medusajs/medusa#2950] [Feature Requests]
As a customer, it would be nice if I could filter and sort products by price. This is currently not in the storefront API, probably because of the complexity behind the price lists, that would make it difficult to build.

**Comment by edihasaj** [comment]
yeah I have the same proposal from customers, this is a good thing to have.

**Comment by luluhoc** [comment]
The sorting by price would be great

**Reply by luluhoc** [reply]
as a workaround I'm adding the price to width and I'm sorting by it

**Comment by mo-hawary** [comment]
We really need this.

**Comment by vholik** [comment]
Try add param to store products: `/store/products?order=-variants.prices.amount` ( `/store/products?order=variants.prices.amount` asc order). Or use search providers like Algolia or Meilisearch

**Reply by alsherif-khalaf** [reply]
I installed Meilisearch how to use it in filtering using Category?

**Reply by vholik** [reply]
I have created a whole guide on how to integrate [Meilisearch with Medusa](https://medium.com/@vholik/complete-guide-on-how-to-integrate-meilisearch-with-medusa-js-13d056836812). There are also examples on how to sort and filter by price, and categories.

**Comment by Loque18** [comment]
this is a very basic feature on every single e-commerce, shoudl be added

**Comment by jerry-rlnk** [comment]
This is indeed a very basic e-commerce function, I'm surprised that Medusa has developed to this point and still doesn't support it, my point is that Medusa is great

**Comment by jalagania** [comment]
This is a must-have feature. Some feedback on this from medusa guys would be nice.
